### PR TITLE
samples: bluetooth: periperal_hr: create minimal configuration

### PR DESCRIPTION
**Pull request by maje-emb**
Create minimal configuration for the peripheral_hr. Which can
be used to build this sample for nRF52810 and nRF52811 targets.

Signed-off-by: Marcin Jeliński <marcin.jelinski@nordicsemi.no>

### CODE REVIEW

**Comment by jhedberg** [comment]
Would this fit on the BBC micro:bit as well? (16k RAM)

**Comment by maje-emb** [comment]
> Would this fit on the BBC micro:bit as well? (16k RAM)

I'm afraid that 16kB RAM might not be enough.  For nrf52810 the RAM usage is around 15kB for this configuration.

File: samples/bluetooth/peripheral_hr/prj_minimal.conf [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+# idle 00             : STACK: unused 208 usage 48 / 256 (18 %); CPU: 97 %
+# main                : STACK: unused 576 usage 448 / 1024 (43 %); CPU: 0 %
+CONFIG_BT_RX_STACK_SIZE=1024
+CONFIG_BT_CTLR_RX_PRIO_STACK_SIZE=448
+CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
+CONFIG_BT_HCI_TX_STACK_SIZE=640
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=512
+CONFIG_IDLE_STACK_SIZE=128
+CONFIG_MAIN_STACK_SIZE=640
+CONFIG_ISR_STACK_SIZE=1024
```
Comment (cvinayak) [inline samples/bluetooth/peripheral_hr/prj_minimal.conf:79]: Did you profile this ISR Stack, could it go down?

Comment (maje-emb) [reply samples/bluetooth/peripheral_hr/prj_minimal.conf:79]: ISR Stack was not profiled. The size was chosen based on other configurations and experience.

**Review by KAGA164 — APPROVED** [review]
Looks good

**Review by joerchan — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Comment by maje-emb** [comment]
@carlescufi Please merge.

**Pull request merged**